docs/astro/src/content/docs/reference/window/contextmenu.mdx

```diff
@@ -0,0 +1,62 @@
+---
+<!-- Copyright © SixtyFPS GmbH <info@slint.dev> ; SPDX-License-Identifier: MIT -->
+title: Window
+description: Window element api.
+---
+import SlintProperty from '/src/components/SlintProperty.astro';
+import Link from '/src/components/Link.astro';
+
+`ContextMenu` is an invisible element that is used to show a context menu.
+
+The context menu is shown if the user right-clicks on the area covered by the `ContextMenu` element,
+or if the user presses the "Menu" key on their keyboard while a `FocusScope` within the `ContextMenu` is focused.
+It is also possible to show the context menu by calling the `show` function on the `ContextMenu` element.
+
+In addition to normal sub elements, the `ContextMenu` element can contain `MenuItem` elements.
+The MenuItem elements are defining the actual menu.
+
+## Function
+
+### show(Point)
+
+Call this function to programmatically show the context menu at the given position relative to the `ContextMenu` element.
```

Comment:
I have the feeling that we might revisit these semantics in the API review ;-)

Reply:
ok, feel free to let me know what feels wrong there.